package.json

```diff
@@ -55,16 +56,19 @@
   "scripts": {
     "build": "vue-cli-service build;cp ./client/public/favicon.ico ./assets",
     "debug": "node debug app.js",
-    "e2e:client": "npx cypress run",
-    "e2e:gui": "npx cypress open",
-    "e2e:server": "npx cypress run --config baseUrl=http://localhost:1337",
+    "dev:devtools": "vue-devtools",
+    "e2e:client": "ENABLE_VUE_DEVTOOLS=true npx cypress run",
+    "e2e:gui": "ENABLE_VUE_DEVTOOLS=true npx cypress open",
+    "e2e:server": "VUE_APP_API_URL=http://localhost:1337 npx cypress run",
     "format:js": "./scripts/format-project-js",
     "lint:fix": "vue-cli-service lint",
     "lint": "vue-cli-service lint --no-fix",
     "start": "node app.js",
-    "start:client": "vue-cli-service serve",
+    "start:client": "ENABLE_VUE_DEVTOOLS=true vue-cli-service serve",
     "start:server": "node app.js",
-    "start:dev": "concurrently \"npm run start:server\" \"npm run start:client\""
+    "start:dev": "concurrently \"npm run start:server\" \"npm run start:client\"",
+    "prod": "npm run build && npm run start:server",
+    "prod:start": "npm run build && npm run start:server"
```

Comment:
These will implicitly get `ENABLE_VUE_DEVTOOLS=false` since the value is not set for these runs.